### ISSUE TITLE
Nel campo "nuovi positivi" sono conteggiati anche "ancora positivi"

### ISSUE THREAD

**Issue by danielesalpietro** [pcm-dpc/COVID-19#1022]
**PREMESSA**
Come sappiamo (e lo dico per esperienza diretta),  il tampone può essere fatto alla stessa persona anche più volte sia quando risulta positivo o negativo; succede sia per il personale sanitario, per verificare che non abbiano contratto il virus, sia per le persone che risultano "positive" dopo alcune settimane per almeno 2 volte (due tamponi negativi); ma anche a persone che per ben 4-5 volte sono risultate **ancora** positive durante tutto il periodo di quarantena (ed a volte anche oltre!).

**PROBLEMA**

Se "**nuovi positivi**"  rappresenta ancora oggi tutti i tamponi "positivi", è probabile che vengano inclusi erroneamente anche gli "ancora positivi", ovvero persone che erano già positive al primo tampone, e che sono "ancora positivi", dopo 2-3-4 tamponi; 

Le persone già positive, sono già contenute nell'insieme "**totale positivi**", non ha senso continuare a conteggiare gli "ancora" positivi dopo il primo tampone "positivo", così si rischia di mal rappresentare la realtà dei fatti (in peggio!).

**SUGGERIMENTO**

Si suggerisce di dividere correttamente i numeri di "**nuovi positivi**" in "**ancora positivi**" e "nuovi positivi" (positivi per la prima volta, quindi realmente solo i nuovi)

**Comment by LucaZeta** [comment]
I tamponi sono etichettati T0 fintanto che le analisi non restituiscono una positività.
In seguito alla positività riscontrata i tamponi per la persona in esame vengono etichettati prograssivamente T1, T2 fino alla doppia negatività consecutiva.

Nel contatore dei nuovi positivi sono considerati i soli tamponi T0.

**Comment by AndreaPlanetGroup** [comment]
Ciao LucaZeta, io sapevo che nel totale positivi finivano tutti, sia i "nuovi positivi" che gli "ancora positivi", più che altro per ladifficoltà logistica nel distinguere i casi.
Quanto tu affermi è documentabile? Sarebbe importante.
Grazie

**Comment by LucaZeta** [comment]
Dal punto di vista epidemiologico sarebbe disastroso quello che dici.
Quanto affermo è comunque in riferimento alle procedure teoriche sempre aperte a possibili errori o deviazioni impreviste.
Sono stati portati e illustrati casi basati su esperienze personali che evidenziavano errori e deviazioni, questo non toglie il principio di base.

No, non ho documenti o procedure da sottoporti, mi spiace.

**Comment by AndreaPlanetGroup** [comment]
Grazie per la risposta!

In base alle statistiche che estrapolo quotidianamente, mi risultava così; se non sbaglio anche la Zampa aveva detto che il problema esisteva e che andava corretto (magari poi è stato fatto).

Comunque grazie

 

Cordiali Saluti,

Andrea Menabue

CEO / CTO

 




  Planet Group srl 

Piazza Unità d’Italia, 1 ||  41053 Maranello (MO)  

Cod. Fisc., P. Iva e Reg. Imp. MO:  03375270364

Tel. 800189484 ||  <http://www.sixerp.it> www.sixerp.it ||  <mailto:info@sixerp.it> info@sixerp.it

 

 

Da: LucaZeta <notifications@github.com> 
Inviato: lunedì 18 gennaio 2021 17:52
A: pcm-dpc/COVID-19 <COVID-19@noreply.github.com>
Cc: AndreaPlanetGroup <andrea.menabue@gestionaleplanet.com>; Comment <comment@noreply.github.com>
Oggetto: Re: [pcm-dpc/COVID-19] Nel campo "nuovi positivi" sono conteggiati anche "ancora positivi" (#1022)

 

Dal punto di vista epidemiologico sarebbe disastroso quello che dici.
Quanto affermo è comunque in riferimento alle procedure teoriche sempre aperte a possibili errori o deviazioni impreviste.
Sono stati portati e illustrati casi basati su esperienze personali che evidenziavano errori e deviazioni, questo non toglie il principio di base.

No, non ho documenti o procedure da sottoporti, mi spiace.

**Comment by LucaZeta** [comment]
> In base alle statistiche che estrapolo quotidianamente, mi risultava così

Per mera curiosità, come pensi di discernere, come li chiami tu, i "nuovi positivi" dagli "ancora positivi"?

Suggerisco di editare il tuo ultimo post, eliminando la parte che non credo volessi pubblicare.

**Comment by AndreaPlanetGroup** [comment]
Grazie del suggerimento, avevo risposto via mail.
Attualmente utilizzo un'approssimazione abbastanza rozza, ma era l'unica che avevo a disposizione: calcolo il rapporto tra i casi testati e i tamponi, che di norma si aggira intorno al 50%, e detraggo quella percentuale dai nuovi positivi.

**Comment by LucaZeta** [comment]
> Attualmente utilizzo un'approssimazione abbastanza rozza, ma era l'unica che avevo a disposizione: calcolo il rapporto tra i casi testati e i tamponi, che di norma si aggira intorno al 50%, e detraggo quella percentuale dai nuovi positivi.

Ho preso un po' di tempo per raccogliere le idee prima di rispondere.

I **casi_testati** rappresentano il numero di persone che si sono sottoposte al test, indipendentemente dall'esito del test di positività o dal numero di test effettuati successivamente.
I **tamponi** rappresentano il numero di test eseguiti sulla popolazione dei casi_testati, indipendentemente dal risultato fornito.

Il loro rapporto, a meno di anomalie, è pari ad 1 ad inizio epidemia, quando ogni test viene fatto ad un nuovo caso, positivo o meno, fino a quando si iniziano a fare test di controllo per verificare eventuali guarigioni di quanti erano risultati positivi. Da quel punto il rapporto inizia a diminuire.

Come e quanto diminuisce e che valore possiamo aspettarci al termine dell'epidemia dal rappporto casi_testati/tamponi?
Tanto maggiore è il numero di test di controllo che vengono effettuati tanto minore sarà il valore del rapporto.
Nei casi_testati rientrano anche i non positivi che possono effettuare il test una sola volta o periodicamente più e più volte.
Impossibile determinare un valore di tendenza a fine epidemia, perchè non dipende dall'epidemia, ma da come questa viene monitorata sulla popolazione.

Per nostra comodità allego il grafico che descrive l'andamento del rapporto casi_testati/tamponi tra i suoi valori totali e giornalieri e in verde, per riferimento, l'andamento giornaliero del numero di tamponi.

![immagine](https://user-images.githubusercontent.com/45511384/105462957-b0ba2500-5c8f-11eb-982c-835b19e06c03.png)

**Comment by AndreaPlanetGroup** [comment]
Grazie per il chiarimento, ritengo sia molto utile per capire meglio questi dati.
Tuttavia rimane in sospeso il fatto che effettivamente i tamponi ripetuti vengano o meno di nuovo conteggiati come positivi; io ho fatto 3 test molecolari, 1 negativo e 2 positivi, ma l'informazione non era presente nè sull'etichetta del campione (barcode incluso), nè sul referto. 
Di conseguenza utilizzo il rapporto casi_testati/tamponi per diminuire il numero di nuovi positivi dichiarato.

**Comment by LucaZeta** [comment]
Così?
E' stato utilizzato il rapporto casi_testati/tamponi totali per calcolare il fattore da applicate all'incidenza giornaliera (Oro) e calcolare quella modificata (Rosso).

In alternativa sarebbe possibile utilizzare il rapporto tra casi_testati e tamponi giornalieri, calcolati per differenza dal giorno precedente.

In termini di valori di incidenza la "correzione" è importante, direi grave.
In termini di andamento e tendenza, direi meno, ma non ho fatto calcoli.

OK. era solo per avere chiaro e comprendere meglio le tue argomentazioni.
Grazie per avermi seguito.

![immagine](https://user-images.githubusercontent.com/45511384/105471487-12cc5780-5c9b-11eb-88ba-eafaa9247de8.png)

**Comment by AndreaPlanetGroup** [comment]
Esatto, io utilizzo la variazione giornaliera quindi quella che nel tuo grafico corrisponde alla linea rossa.
Concordo con la tua analisi, in termini di valori assoluti la correzione è importante e quindi grave; nulla cambia dal punto di vista del trend, ovviamente.
Sarebbe a questo punto fondamentale conoscere con certezza questa informazione relativa al conteggio o meno dei tamponi ripetuti.
Grazie a te!!

**Comment by LucaZeta** [comment]
> Sarebbe a questo punto fondamentale conoscere con certezza questa informazione relativa al conteggio o meno dei tamponi ripetuti.

Considero il conteggio dei tamponi ripetuti positivi un errore di procedura e come tale diffilile o impossibile da quantificare.
Personalmente lo considero un numero trascurabile, ma potresti tranquillamente avere ragione tu.

**Comment by AndreaPlanetGroup** [comment]
Confido di riuscire ad avere questa informazione, visto che con un pò di perseveranza sono riuscito ad ottenere altri dati apparentemente non  disponibili (come le sequenze genetiche rilevate nei miei tamponi, i cicli di amplificazione ecc.).
Ovviamente in caso di successo condividerò l'informazione!

**Comment by AndreaPlanetGroup** [comment]
![image](https://user-images.githubusercontent.com/15998070/111458665-005e1f80-871a-11eb-9286-7b95a982b592.png)

**Comment by davideandrei** [comment]
@danielesalpietro sono d'accordo con la tua proposta.

Penso però che anche se si facesse questa cosa
(A mio parere giusta) non cambierebbe il numero di decessi e di persone in TI che sono i dati più rilevanti... se tutti dopo 2 settimane curavano il Covid-19 a casa senza farmaci rilevanti allora non saremo quasi tutti in zona rossa e, io, da studente, in DAD
